subt_ign/src/GameLogicPlugin.cc

```diff
@@ -2580,7 +2580,7 @@ void GameLogicPluginPrivate::LogRobotArtifactData(
   if (!this->foundArtifacts.empty())
   {
     meanReportTime = (this->lastReportTime-this->firstReportTime) /
-        this->foundArtifacts.size();
+        (this->foundArtifacts.size() - 1);
```

Comment:
Can you add a check that this won't divide by 0?

Reply:
Added in a2213c2112a958273299fc8660810793c10434c7